pkg/trace/config/apply_test.go

```diff
@@ -59,3 +63,62 @@ func TestSplitTag(t *testing.T) {
 		})
 	}
 }
+
+// TestSQLObfuscationConfigDeserializationMethod checks if the use of easyjson results in the same deserialization
+// output as encoding/json.
+func TestSQLObfuscationConfigDeserializationMethod(t *testing.T) {
+	mockSQLConfig := SQLObfuscationConfig{QuantizeSQLTables: true}
+	mockSQLConfigBytes, err := json.Marshal(mockSQLConfig)
+	require.NoError(t, err)
+
+	var regularJSONDeserialized, easyJSONDeserialized SQLObfuscationConfig
+
+	err = json.Unmarshal(mockSQLConfigBytes, &regularJSONDeserialized)
+	require.NoError(t, err)
+
+	jl := &jlexer.Lexer{Data: mockSQLConfigBytes}
+	easyJSONDeserialized.UnmarshalEasyJSON(jl)
+	require.NoError(t, jl.Error())
+
+	assert.Equal(t, regularJSONDeserialized, easyJSONDeserialized)
+}
+
+func BenchmarkSQLObfuscationConfigEasyJSONDeserialization(b *testing.B) {
+	for i := 1; i <= 100000; i *= 10 {
+		b.Run(fmt.Sprintf("Range:%d", i), func(b *testing.B) {
+			benchmarkSQLObfuscationConfigEasyJSONDeserialization(b)
+		})
+	}
+}
+
+func benchmarkSQLObfuscationConfigEasyJSONDeserialization(b *testing.B) {
+	b.ReportAllocs()
+	mockSQLConfig := SQLObfuscationConfig{QuantizeSQLTables: true}
+	mockSQLConfigBytes, err := json.Marshal(mockSQLConfig)
```

Comment:
Please do the same here. Variable names are way too long.

Reply:
Updated: https://github.com/DataDog/datadog-agent/pull/8556/commits/209c8e671c26c61ab3ef64fe3b5e5adf7179fdc0